Content.Client/Preferences/UI/HumanoidProfileEditor.xaml.cs

```diff
@@ -1125,6 +1083,8 @@ private void UpdatePreview()
 
             if (ShowClothes.Pressed)
                 LobbyCharacterPreviewPanel.GiveDummyJobClothes(_previewDummy!.Value, Profile);
+
+            _previewSpriteView.OverrideDirection = (Direction) (_previewRotation % 4 * 2);
```

Comment:
whats the % 4 * 2 for?

Reply:
should be commented but it seems to be for wrapping + ignoring diagonal directions

Reply:
This is redundant already. Removed

Reply:
Yes, this to ignore diagonal directions